src/pkcs11_1000.c

```diff
@@ -0,0 +1,7971 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright (c) 2018, Linaro Limited
+ * Copyright (c) 2023, EPAM Systems
```

Comment:
it is sad it isn't possible to view changes that were added by you inside this pull request
the same story about other ported files

Reply:
What do you mean under term "view changes"? If you expect to see original file as a separate commit - that's not a good idea because it will make optee-test unbuildable on this commit. This file, similar to others, was adapted to work with ztest. No functional changes were done